enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md

```diff
@@ -84,9 +84,9 @@ Logs of a given type are co-located to the following indices:
 
 | Log Type | Read Alias | Write Alias | Initial Index |
 |---------|-----------|---------------|---------------|
-| Infra (`logs.infra`)|infra,logs.infra|infra-write|infra-00001|
-| Application Container (`logs.app`)|app,logs.app|app-write|app-000001|
-| Audit (`logs.audit`)|infra.audit,logs.audit|infra.audit-write|audit.infra-000001|
+| Infra (`logs-infra`)|infra,logs-infra|infra-write|infra-00001|
```

Comment:
Same here for "Log Type", these can remain dots pending input from @alanconway regarding what we should call them to jive with LogForwarder

Reply:
See https://github.com/openshift/enhancements/pull/241#discussion_r390614769